src/nginx/module.cc

```diff
@@ -177,6 +177,9 @@ ngx_esp_request_ctx_s::ngx_esp_request_ctx_s(ngx_http_request_t *r,
       if (lc->esp->get_always_print_primitive_fields()) {
         json_print_options.always_print_primitive_fields = true;
       }
+      if (lc->esp->get_preserve_proto_field_names()) {
+        json_print_options.preserve_proto_field_names = true;
```

Comment:
So, the transcoding lib does support this feature, what needs to be done here is just pass in the configuration?

Should we do this for CloudESF as well?

Reply:
Yes